main.go

```diff
@@ -1,3 +1,144 @@
 package main
 
-func main() {}
+import (
+	"fmt"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/microkit/command"
+	microserver "github.com/giantswarm/microkit/server"
+	"github.com/giantswarm/microkit/transaction"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/microstorage"
+	"github.com/giantswarm/microstorage/memory"
+	"github.com/giantswarm/node-operator/flag"
+	"github.com/spf13/viper"
+
+	"github.com/giantswarm/node-operator/server"
+	"github.com/giantswarm/node-operator/service"
+)
+
+var (
+	description string     = "Project node-operator drains Kubernetes nodes on behalf of watched CRDs."
+	f           *flag.Flag = flag.New()
+	gitCommit   string     = "n/a"
+	name        string     = "node-operator"
+	source      string     = "https://github.com/giantswarm/node-operator"
```

Comment:
I don't know if it's important for us, but `golint` would give you a warning about unnecessary types here.
> main.go:26:14: should omit type string from declaration of var source; it will be inferred from the right-hand side

I stripped them out in `cluster-operator` in order to get clean `golint` run.

Reply:
Good point. Removed.